.github/workflows/linters.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup Webhint
         run: |
           npm install --save-dev hint@6.x
-          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
       - name: Webhint Report
         run: npx hint .
```

Comment:
- [OPTIONAL] I see you have a Linters problem, I cloned your repo and run `npx hint .` and it's passing the test. I would recommend uploading a screenshot to your PR when this happens, this way the code reviewers can see that is a GitHub error and won't invalidate your review.

![Captura de pantalla 2022-06-14 a la(s) 10 50 56 p  m](https://user-images.githubusercontent.com/83717285/173733239-6c3b9023-1d9f-4ab3-aa16-23fbcb150704.png)